### PR TITLE
Include Compose by default

### PR DESCRIPTION
**Pull request by idyatlov**
**Description**:

- Use Compose by default, remove -compose suffix
- Update Compose to 1.0 and update required dependencies

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,9 +23,9 @@ jobs:
                 path: |
                   ~/.gradle/caches
                   ~/.gradle/wrapper
-                key: ${{ runner.os }}-gradle-false-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+                key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
                 restore-keys: |
-                  ${{ runner.os }}-gradle-false-
+                  ${{ runner.os }}-gradle-
```
Comment (CherryPerry) [inline .github/workflows/ci.yml:28]: Remove this line, the next one is the same.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -80,18 +71,18 @@ jobs:
                 path: |
                   ~/.gradle/caches
                   ~/.gradle/wrapper
-                key: ${{ runner.os }}-gradle-${{ matrix.compose-enabled }}-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+                key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
                 restore-keys: |
-                  ${{ runner.os }}-gradle-${{ matrix.compose-enabled }}-
+                  ${{ runner.os }}-gradle-
```
Comment (CherryPerry) [inline .github/workflows/ci.yml:76]: Remove

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -124,22 +111,22 @@ jobs:
...
```
Comment (CherryPerry) [inline .github/workflows/ci.yml:116]: Remove

**Review by CherryPerry — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 3 above)
Comment (idyatlov) [reply .github/workflows/ci.yml:114]: Done

File: .github/workflows/ci.yml (hunk 2 above)
Comment (idyatlov) [reply .github/workflows/ci.yml:75]: Done

File: .github/workflows/ci.yml (hunk 1 above)
Comment (idyatlov) [reply .github/workflows/ci.yml:28]: Done

**Review by CherryPerry — APPROVED** [review]

**Pull request merged**